zcash_client_sqlite/src/error.rs

```diff
@@ -3,18 +3,22 @@
 use std::error;
 use std::fmt;
 
+use nonempty::NonEmpty;
 use shardtree::error::ShardTreeError;
+
 use zcash_address::ParseError;
 use zcash_client_backend::data_api::NoteFilter;
+use zcash_keys::address::UnifiedAddress;
 use zcash_keys::keys::AddressGenerationError;
-use zcash_protocol::{consensus::BlockHeight, value::BalanceError, PoolType};
+use zcash_protocol::{consensus::BlockHeight, value::BalanceError, PoolType, TxId};
+use zip32::DiversifierIndex;
 
 use crate::{wallet::commitment_tree, AccountUuid};
 
 #[cfg(feature = "transparent-inputs")]
 use {
-    ::transparent::address::TransparentAddress, zcash_keys::encoding::TransparentCodecError,
-    zcash_primitives::transaction::TxId,
+    crate::wallet::KeyScope, ::transparent::address::TransparentAddress,
+    zcash_keys::encoding::TransparentCodecError, zip32::Scope,
```

Comment:
```suggestion
    ::transparent::{address::TransparentAddress, ::keys::TransparentKeyScope},
    zcash_keys::encoding::TransparentCodecError, zip32::Scope,
```

Reply:
Included in 0dcb2b2e74f6d8f0552c6cab01e2e0061150b3ce